### PR TITLE
Deprecate set_and_verify api

### LINKED ISSUE #208: Deprecate Set_And_Verify API
(This Issue started as a discussion on: Rename variorum_cap_and_verify_node_power_limit as variorum_cap_and_verify_best_effort_node_power_limit ?, so some notes may be irrelevant).

We had decided to make everything at the node-level best effort, given the differences in platforms. The two APIs should be consistent. I believe the cap_and_verify is only currently implemented for IBM (need to check). Creating an issue so this is logged somewhere.

### PR DESCRIPTION
**Pull request by tpatki**
Fixes #208

### CODE REVIEW

**Comment by slabasan** [comment]
This is deprecating cap_and_verify, not set_and_verify, right?

File: src/variorum/IBM/config_ibm.c [hunk 1]
```diff
@@ -28,12 +28,10 @@ int set_ibm_func_ptrs(void)
         g_platform.variorum_print_power = p9_get_power;
         g_platform.variorum_print_power_limits = p9_get_power_limits;
         g_platform.variorum_cap_best_effort_node_power_limit =
-            p9_cap_node_power_limit;
+            p9_cap_and_verify_node_power_limit;
```
Comment (slabasan) [inline src/variorum/IBM/config_ibm.c:31]: I thought cap_and_verify was deprecated in this PR?

**Review by slabasan — COMMENTED** [review]
@tpatki Couple questions for you regarding set and verify vs cap and verify. Cap and verify function still exists in this PR.

File: src/variorum/IBM/config_ibm.c (hunk 1 above)
Comment (tpatki) [reply src/variorum/IBM/config_ibm.c:31]: This is the internal function for IBM which I didn't rename. Happy to rename it if you recommend. As we had discussed, we still want to verify that the cap was set, as a delay can be encountered sometimes.

File: src/variorum/IBM/Power9.c [hunk 2]
```diff
@@ -204,8 +204,10 @@ int p9_cap_and_verify_node_power_limit(int pcap_new)
 
     if (pcap_new != pcap_test)
     {
-        fprintf(stdout, "Could not verify if the power cap was set correctly.\n");
-        fprintf(stdout, "Verification check after 100ms failed.\n");
+        fprintf(stdout,
+                "IBM systems may encounter a delay when setting power limits on the node.");
+        fprintf(stdout, "We could not verify if the power cap was set correctly.\n");
+        fprintf(stdout, "The verification check after 100ms failed.\n");
         fprintf(stdout, "Please verify again with print_power_limits.\n");
```
Comment (slabasan) [inline src/variorum/IBM/Power9.c]: This should change to match current API `print_power_limit`. Also should this be `variorum_print_power_limit()`?

File: src/variorum/IBM/config_ibm.c (hunk 1 above)
Comment (slabasan) [reply src/variorum/IBM/config_ibm.c:31]: Ah ok, understand now.

**Pull request merged**